ts/runtime-asyncify.ts

```diff
@@ -64,6 +61,19 @@ export class QuickJSAsyncRuntime extends QuickJSRuntime {
     return context
   }
 
+  private getNewContextPointer(options: ContextOptions = {}): JSContextPointer {
```

Comment:
What's the difference from `getNewContextPointer` in the parent class?

Reply:
The parent class allows for recycling of an already existing context, which didn't seem to be an existing feature of the async runtime